example/properties_directory/structural/molecular_weight.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "molecular_weight",
+    "value": 16.0313,
+    "units": "g/mol"
+}
```

Comment:
Where are the changes to the symmetry property with `pointGroupSymbol` etc??

Reply:
point group is already included in symmetry https://github.com/Exabyte-io/esse/blob/dev/schema/properties_directory/structural/symmetry.json

Reply:
**NEW UPDATE** separated point group and space group into two separate schema instead of keeping them together inside a broader `symmetry` schema.